test_app/server.R

```diff
@@ -19,13 +19,13 @@ function(input, output, session) {
     validation <- TRUE
   })
   
-  # 3. Hide form (in case credentials are correct) - ----
+  # 2. Hide form (in case credentials are correct) and Update log table
   observeEvent(validate_password_basic(), {
     shinyjs::hide(id = "login-basic")
+    UpdateLog(input$ti_user_basic)
```

Comment:
It works but if the user does a wrong attempt to login the message still appear for 1 second once logged-in

Reply:
I managed to do something maybe not an optimal solution, but:

the validation_password function creates an object of class `shiny-output-error-validation` automatically when hitting validation, for so I set the size of it to 0. 
Then i created a new output id with a div, and instead use the class `shiny-output-error-validation`, I used `shiny-text-output`  (see lines from 41 to 57 in the `ui` function)

Reply:
I managed to do something which seems to do the job and I particularly like:

the valide_login function creates an object of class `shiny-output-error-login` when it fails. In order to print only the error and not output (which is automatically triggered as well) creating a double error message, I created a reactive value, which is FALSE until the validation fail and become TRUE when validation pass. 
Then i created a new output id with a div, and used both the classes `shiny-text-output` (because in the end it is a text) and `text warning` (in order to inherit the emphasis class from the bootstrap theme)